tests/test_scp_client.cpp

```diff
@@ -123,16 +123,22 @@ TEST_F(SCPClient, throws_on_push_file_invalid_source)
 
 TEST_F(SCPClient, throws_when_pull_file_request_not_newfile)
 {
+    mpt::TempDir temp_dir;
+    auto file_name = temp_dir.path() + "/test-file";
```

Comment:
Does the scp client handle the forward slash on windows?

Reply:
I suppose so, as I see this in the code... https://github.com/CanonicalLtd/multipass/blob/4a9707473e1104ecb2d406fb37b0ebb469cfdbee/src/ssh/scp_client.cpp#L55

Reply:
@ricab,

Yes, it's fine on Windows as other tests already use forward slashes and it works without issue.

Reply:
ok, approving then